### ISSUE TITLE
gt label的问题

### ISSUE THREAD

**Issue by elvindp** [OpenFirework/pytorch_solov2#10]
在'solov2_head.py'里，'solov2_target_single'函数会生成ground truth label，
cate_label初始值是0：
cate_label = torch.zeros([num_grid, num_grid], dtype=torch.int64, device=device)
然后在下边根据中心点的位置，更新cate_label:
cate_label[top:(down+1), left:(right+1)] = gt_label
但class是从0-80，第0类是人，物体很多时候并不是布满整幅图的，这样会cate label里就有大量的0类
有人可以帮忙解释一下不？谢谢

**Comment by ViatorSun** [comment]
第0类应该是背景吧，在config.py 文件里面有映射，person 对应标签1
